src/modules/profile/views/waitingTimeStatistics/waitingTimeStatistics.tsx

```diff
@@ -0,0 +1,89 @@
+import { ArcElement, Chart as ChartJS, type ChartOptions, Legend, Tooltip } from 'chart.js';
+import { useEffect, useMemo, useState } from 'react';
+import { Pie } from 'react-chartjs-2';
+
+ChartJS.register(ArcElement, Tooltip, Legend);
+
+interface statistics {
```

Comment:
The naming convention for TypeScript interfaces typically starts with an uppercase letter.
So, interface `statistics` should be interface `Statistics`.